.github/workflows/ci.yml

```diff
@@ -249,21 +249,10 @@ jobs:
       - run: nix flake check
         continue-on-error: true
 
-  check-links:
-    runs-on: ubuntu-24.04
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          show-progress: false
-      - uses: lycheeverse/lychee-action@v2.0.2 # later versions break fragment checks. don't bump until this is fixed: https://github.com/lycheeverse/lychee/issues/1574
```

Comment:
What about this btw? Failing the CI if links are broken is useful

Reply:
I have enabled strict mode in the mkdocs config. This will fail the build if links are broken by itself, rendering lychee superfluous imo :)